packages/matchbox/src/components/Snackbar/Snackbar.js

```diff
@@ -15,6 +16,7 @@ const StyledBox = styled(Box)`
 `;
 
 const StyledClose = styled(Box)`
+  ${buttonReset}
```

Comment:
Without this, there was a visual bug that prevented the icons from using the correct color